service/matter.go

```diff
@@ -36,6 +36,19 @@ func MatterParentExist(uid int64, parentDir string) bool {
 	return false
 }
 
+func MatterDirFileNum(uid int64, dir string) (num int64) {
+	result := gormutil.DB().Where("uid=? and parent=?", uid, dir).Find(&model.Matter{})
+	return result.RowsAffected
```

Comment:
Why not use the `Count`? 

`Find` will return all rows of data. It will lead to a long read time when the amount of data is large